meta/style_guide.md

```diff
@@ -0,0 +1,45 @@
+# Style Guide
+
+What follows are style guide requirments for the rule book project. These are conventions we should attempt to follow, but they are not hard and fast rules like the [syntax document](syntax.md).
+
+## Terms and Definitions
+
+There are 3 styles to provide terms and definitions.
+
+For simple sets of words, follow a modified *Associated Press* (AP) style. Provide the full name, like "Associated Press" in the previous sentence, as italic followed by an acronym or shortened name in parentheses.
+
+For terms that can be described in a sentence, use Markdown's "Terms and Definitions" as defined in the [syntax document](syntax.md).
+
+For terms that require lengthier definitions, write the term as a header and description as paragraphs below it.
+
+## Reserved Header Names
+
+There are header names that are reserved for special purposes:
+
+- Application
+- Commentary
+- Examples
+- Scoring
+
+At any given normal header level (and thus any given spot within a document's information architecture), providing any of the reserved header names indicates that any content within that header is considered tagged by the reserved name such that processing of the source can selectively include or exclude that section.
```

Comment:
I'm a bit unclear on the specific use of these reserved headers. Please let me know if I'm dense, or if the documentation can improve. When using a reserved header, should the header only include the specific word, and nothing else? Which means that the specifics on (e.g. what specifically related to Scoring I'm talking about) would go below the header?

Reply:
The documentation can improve. :)